.circleci/config.yml

```diff
@@ -73,4 +73,4 @@ workflows:
                       branches:
                           ignore: /.*/
                       tags:
-                          only: /chart\/kafka-operator\/\d+.\d+.\d+/
+                          only: /chart\/kafka-operator\/\d+.\d+.\d+(-rc\.\d+)?/
```

Comment:
With the change, will the helm chart release versions without `-rc.x` suffix work?

Reply:
Yep this regex has an optional field in the end `(-rc\.\d+)?` it means this field can be omitted.